src/harps-n/io.jl

```diff
@@ -0,0 +1,142 @@
+"""
+   IO functions for the HARPS-N spectrograph
+    https://plone.unige.ch/HARPS-N/
+Author: Alex Wise and collaborators
+Created: April 2021
+"""
+
+"""Create Dataframe containing filenames and key data for all files YYYY-MM-DDThh:mm:ss.mss (year-month-day T hour:minute:second:milisecond in directory"""
+function make_manifest(data_path::String ; max_spectra_to_use::Int = 1000 )
+    df_filenames = EchelleInstruments.make_manifest(data_path,r"^20\d{2}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}$")
+
+    df_files = DataFrame(read_metadata(joinpath(df_filenames.Filename[1],"s2d.fits")))
+    keys = propertynames(df_files)
+    allowmissing!(df_files, keys[map(k->k∉[:Filename, :bjd, :target, :airmass],keys)] )
+
+    if length(df_filenames.Filename) >= 2
+        map(fn->add_metadata_from_fits!(df_files,joinpath(fn,"s2d.fits")),df_filenames.Filename[2:end])
+    end
+
+    RV_data = CSV.read(joinpath(data_path,"harpn_sun_release_timeseries_2015-2018.csv"), DataFrame)
+
+    # berv = RV_data[:, [:berv]]
```

Comment:
These look like they're specific to solar observations.  So I'm worried it might break for non-solar observations.  Should we make separate functions for night and solar observations?  Or add a function parameter to specify whether night or solar?

Reply:
make filename a param, if it == none, do nothing, if doesn't exist add warning message